ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -253,37 +220,19 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
     return state;
   }
 
-  get usernameValid(): boolean {
+  // Returns whether the current username is considered valid. Undefined is returned when the
+  // username is empty, or if a username check is in progress.
+  isUsernameValid(): (boolean|undefined) {
```

Comment:
Why remove the get on this as well? Once again, shouldn't have any side effects?